ci/test-footer.sh

```diff
@@ -0,0 +1,4 @@
+trap - EXIT SIGHUP SIGINT SIGTERM
+
+# If we got here, then build passed
+echo -e "\nBuild successful"
```

Comment:
The echo is misleading and can be confused with the one in `start.sh`.

If I understood correctly this is only the result of the test, not the "whole" build.